modules/rostests/apitests/user32/CharFuncs.c

```diff
@@ -0,0 +1,597 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Tests for Char* functions
+ * COPYRIGHT:   Copyright 2022 Stanislav Motylkov <x86corez@gmail.com>
+ */
+
+#include "precomp.h"
+
+#include <strsafe.h>
+#include <windows.h>
+#include <ntndk.h>
```

Comment:
ntndk for which function(s)?

Reply:
`ntndk.h` is for `STATUS_SUCCESS` and co.

Reply:
```c
#include <ntstatus.h>
#define WIN32_NO_STATUS
// Rest of the PSDK here.
```

Reply:
Ok I have refactored headers.

Reply:
`#include <ntstatus.h>` doesn't seem to work: `../modules/rostests/apitests/user32/CharFuncs.c:202:5: error: unknown type name 'NTSTATUS'`

Reply:
Another possibility that seems to work:
1. in the test' precomp header, ensure that `#define WIN32_NO_STATUS` is defined before any PSDK includes.
2. in your CharFuncs.c, after the last psdk include (winnls), do `#undef WIN32_NO_STATUS`, and then include ntstatus.h.

Reply:
`#define WIN32_NO_STATUS` is already defined in `precomp.h`. Then, adding undef and ntstatus.h in `CharFuncs.c` results in very same error.